dev/io.openliberty.checkpoint/resources/io/openliberty/checkpoint/resources/CheckpointMessages.nlsprops

```diff
@@ -37,3 +37,22 @@ RESTORE_FAILED_CWWKC0454E=CWWKC0454E: The server failed to restore from a checkp
 RESTORE_FAILED_CWWKC0454E.explanation=An attempt to restore the server from a checkpoint failed.  The server is not running.
 RESTORE_FAILED_CWWKC0454E.useraction=Inspect the checkpoint directory located in the server logs directory for additional log files to determine why the server could not be restored.
 
+UKNOWN_FAILURE_CWWKC0455E=CWWKC0455E: An unknown error occurred with the following message: {0}
+UKNOWN_FAILURE_CWWKC0455E.explanation=An unexpected error occurred during a checkpoint request or during an attempt to restore the server from a checkpoint.
+UKNOWN_FAILURE_CWWKC0455E.useraction=Inspect the checkpoint directory located in the server logs directory for additional log files to determine why the error occurred.
```

Comment:
The useraction wording is okay. To shorten it, you might use "Inspect the log files in the checkpoint directory that is in the server logs directory to determine why the error occurred."

Reply:
I was copying from previously approved useraction messages for CWWKC0453E and CWWKC0454E.  If we do shorten I would do it for all cases like this.